dc-model-feign/src/main/java/de/digitalcollections/model/feign/encoder/filter/FilteringToQueryMapEncoder.java

```diff
@@ -0,0 +1,78 @@
+package de.digitalcollections.model.feign.encoder.filter;
+
+import de.digitalcollections.model.api.filter.FilterCriteria;
+import de.digitalcollections.model.api.filter.Filtering;
+import de.digitalcollections.model.api.filter.enums.FilterOperation;
+import feign.QueryMapEncoder;
+import java.time.LocalDate;
+import java.time.chrono.ChronoLocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+public class FilteringToQueryMapEncoder<T extends Comparable> implements QueryMapEncoder {
+
+  @Override
+  public Map<String, Object> encode(Object object) {
+    if (object == null) {
+      return null;
+    }
+    Map<String, Object> queryMap = new HashMap<>();
+
+    Filtering filtering = (Filtering) object;
+    for (Iterator iterator = filtering.iterator(); iterator.hasNext(); ) {
```

Comment:
`for (FilterCriteria<T> : filtering)` geht hier nicht?

Reply:
stimmt. done

Reply:
done